### PR TITLE
Add solar panels and storage batteries to the power grid

### PR DESCRIPTION
**Pull request by Fris0uman**
#### Summary
Content "Add solar panels and storage batteries to the power grid"

#### Purpose of change

Complete and replace  #54111

Part of #53647 

I don't how to push directly to the open PR so I made a new one
#### Describe the solution
Use existing storage batteries and solar panel entries to make the appliances one, thanks to #55042

#### Testing

image

image

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-appliances.json [hunk 1]
```diff
@@ -571,17 +571,7 @@
     "move_cost_mod": 2,
     "required_str": -1,
     "flags": [ "TRANSPARENT" ],
```
[the remark this answers is not in the document]
Comment (Fris0uman) [reply data/json/furniture_and_terrain/furniture-appliances.json:573]: I think unmounting a solar panel should require some tool. It's not like the oven or the frigde that you just pick up, solar panels are probably bolted down

**Comment by Xenoamor** [comment]
I think the solar panel and battery item descriptions will likely need updating after this change

File: data/json/furniture_and_terrain/furniture-appliances.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Xenoamor) [reply data/json/furniture_and_terrain/furniture-appliances.json:573]: Yes, that does make sense!

**Comment by Fris0uman** [comment]
What would you change? it can change as part of this PR

**Comment by Xenoamor** [comment]
Probably just scrubbing the vehicle references in "data/json/items/vehicle/solar.json" would be enough
"Electronic device that can convert solar radiation into electric power. ~~Useful for a vehicle.~~"

The battery ones might be okay as it stands actually. We may wish to add the other solar panel variants however:

- reinforced solar panel
- upgraded solar panel
- upgraded reinforced solar panel

**Pull request merged**